onecodex/viz/_heatmap.py

```diff
@@ -238,6 +251,24 @@ def plot_heatmap(
                 haxis, header=alt.Header(titleOrient="bottom", labelOrient="bottom")
             )
 
+        # Drop all-NaN rows, convert remaining NaNs to 0s, and concat
+        # dropped all-NaN rows to the end
+        dropped = []
+        for classification_id in all_nan_classification_ids:
+            d = df[(df == classification_id).any(axis=1)]
```

Comment:
To get a specific row in a dataframe by index label:

```python
row = df.loc[classification_id]
```

Reply:
I'm getting `KeyError` with that

Reply:
oh my bad, I thought the rows in the dataframe were classification IDs, and the columns were taxa, but that's only the case when `to_df(table_format="wide")` is used (the format is `"long"` here)